### DISCUSSION TITLE
E-mail markdown panel still doesn't work

### DISCUSSION THREAD

**Discussion by preliot** [laravel/framework#32884] [General] [answered]
- Laravel Version: 7.11.0
- PHP Version: 7.2.19
- Database Driver & Version: mysql 5.7.24

### Description:

E-mail markdown for panels seems to be broken. I use panels to make information standout, so i really need them. I've read several other related issues and tried all solutions, but it didn't work.

I've started to upgrade my application to laravel 7.x (was on 5.8) when i came accross this issue.  Tried to deintend everything (that did fix some headers). Tried to add empty lines (before, after, everywhere), but that didn't seem to work. 

Removing the panel is the only thing that works (then i can see a table without html). 

### Steps To Reproduce:

Create template:

```php
@component('mail::message')
@component('mail::panel')
@component('mail::table')
| a   | b   | c   |
| --- | --- | --- |
1|2|3
@endcomponent
@endcomponent
@endcomponent
```

Results in this:
![image](https://user-images.githubusercontent.com/5418196/82298759-e302dc80-99b4-11ea-9f73-ff400c93d5cd.png)

### Question?

How can i fix my template to get panels to work?

**Comment by nmokkenstorm** [comment]
You might just want to raise an issue for this in the issue tracker.

**Reply by preliot** [reply]
Isn't this the issue tracker?

**Reply by driesvints** [reply]
I've moved this to discussions instead because it's more a support question than a bug.

**Comment by driesvints** [comment, marked as answer]
Your markdown is invalid.

Try

```blade
@component('mail::message')
@component('mail::panel')
@component('mail::table')
| a   | b   | c   |
| --- | --- | --- |
|1|2|3
@endcomponent
@endcomponent
@endcomponent
```

**Reply by preliot** [reply]
I've copied this markdown literaly, still gives the same result. Also experimented with empty lines, adding / removing | around tables. Only thing working is removing mail::panel component

**Reply by GrahamCampbell** [reply]
```
| a   | b   | c   |
| --- | --- | --- |
|1|2|3|
```
I think is what Dries meant?

**Reply by preliot** [reply]
Unfortunatly the same result.

**Reply by driesvints** [reply]
Did you try a white line between the components?

```blade
@component('mail::message')
@component('mail::panel')
@component('mail::table')

| a   | b   | c   |
| --- | --- | --- |
|1|2|3|

@endcomponent
@endcomponent
@endcomponent
```

**Reply by preliot** [reply]
Copied & pasted you example. Same result. Browser inspection reveals `<code>` tag:

```html
<pre style="font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; box-sizing: border-box;"><code>                &lt;/td&gt;
            &lt;/tr&gt;
        &lt;/table&gt;
    &lt;/td&gt;
&lt;/tr&gt;
</code></pre>
```

**Reply by driesvints** [reply]
I think at this point it's best that you provide an example app that reproduces the problem. Please commit all custom changes in a separate commit from the base Laravel installation.

**Reply by preliot** [reply]
Created a fresh laravel install and ... it worked like expected. There must be something wrong in my code, packages or nova. Thanks for your efforts!

**Reply by preliot** [reply]
`php artisan vendor:publish --tag=laravel-mail --force`

Worked with old templates, forgot the force flag. Fixed!
